source/js/components/highlights/highlights.jsx

```diff
@@ -9,37 +9,44 @@ export default class Highlights extends React.Component {
   }
 
   render() {
-    // Limit to 2 additional highlights
-    let highlights = this.props.data.slice(1, 3).map((item, index, array) => {
+    let superHighlights = this.props.data.slice(0, 2).map((item, index) => {
```

Comment:
should we create a `const` for `2`?

Reply:
I mean...we could, but it's always going to be 2.